shell/platform/android/io/flutter/embedding/android/FlutterFragment.java

```diff
@@ -757,8 +767,10 @@ public FlutterFragment() {
   // TODO(mattcarroll): remove this when tests allow for it
   // (https://github.com/flutter/flutter/issues/43798)
   @VisibleForTesting
-  /* package */ void setDelegate(@NonNull FlutterActivityAndFragmentDelegate delegate) {
-    this.delegate = delegate;
+  /* package */ void setDelegateFactory(
```

Comment:
Remove todo and close issue if not relevant anymore

Reply:
This issue still seems to apply.

Reply:
The issue itself is to remove the method you're removing here. We still need some way to test this class and this seems like an appropriate way to have some DI.